### PR TITLE
added more global styles and the transform magic to make the main por…

### PR DESCRIPTION
**Pull request by Nolan-Little**
…trait spin. 

1. Link and content font color: darker blue  #143e6e;
2.top nav and footer padding: added padding for more consistent sizing
3.content borders and border radii: more visually pleasing content divisions with rounded borders
4. content title padding and margin: less claustrophobic text. 
5. content padding: see last item.

6. added brads transform magic. 

take special note. there's a lot of hodor in there as a placeholder.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -12,8 +12,9 @@ body {
   position: fixed;
   width: 100vw; 
   background-color: #EAC4D5;
-  color: #56789F;
   top: 0px;
+  padding: 8px;
+  z-index: -2;
```
Comment (braddavistech) [inline style.css]: This should just be "z-index: 2;" so that the nav bar is always on top. Other than that, looks good. Nice job.

Comment (Nolan-Little) [reply style.css]: fixed in latest push!

**Comment by braddavistech** [comment]
✔️ 👍

File: style.css [hunk 2]
```diff
@@ -51,11 +53,12 @@ a {
   display: flex;
   flex-flow: row nowrap;
   justify-content: center;
-  margin: 50px;
+  margin: 75px;
```
Comment (jasehackman) [inline style.css:56]: Would it look better if the picture had even spacing above and below? if you add margin-top: 120px I thought it looked pretty balanced.

**Comment by jasehackman** [comment]
Had one suggested change to the css to balance the space above and below the image by adding margin-top:120px on the image.  Just a suggestion. If you disagree that fine with me. Beyond that it gets a 👍

**Comment by danielreedcombs** [comment]
👍

**Comment by jasehackman** [comment]
perfect!

**Pull request merged**